src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -56,6 +56,15 @@ public ResponseEntity<List<ModificationInfos>> getNetworkModifications(@Paramete
         return ResponseEntity.ok().body(networkModificationService.getNetworkModifications(groupUuid, onlyMetadata, errorOnGroupNotFound));
     }
 
+    @GetMapping(value = "/groups/{groupUuid}/modifications-restore", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
You could use the endpoint just above. And add a parameter 'stashedModification'

Reply:
parameter added